<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1040</width>
    <height>418</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CyDownload by thegreatjoj</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QComboBox" name="cmb_repo_urls">
      <property name="editable">
       <bool>false</bool>
      </property>
      <property name="currentText">
       <string/>
      </property>
      <property name="currentIndex">
       <number>-1</number>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="btn_open_repo">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Ouvrir un repo</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_del_repo">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">border: 2px solid #e50000;</string>
        </property>
        <property name="text">
         <string>Supprimer un repo</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="btn_new_repo">
      <property name="text">
       <string>Nouveau repo</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPlainTextEdit" name="txt_log">
      <property name="acceptDrops">
       <bool>false</bool>
      </property>
      <property name="undoRedoEnabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QProgressBar" name="progressBar">
      <property name="value">
       <number>0</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <action name="actionOpen_Repo">
   <property name="text">
    <string>Open Repo</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
